ipwhois/tests/test_net.py

```diff
@@ -16,23 +16,35 @@ class TestNet(TestCommon):
     def test_ip_invalid(self):
         self.assertRaises(ValueError, Net, '192.168.0.256')
         self.assertRaises(ValueError, Net, 'fe::80::')
+        self.assertRaises(ValueError, Net, '192.256.0.0/16')
+        self.assertRaises(ValueError, Net, 'fe::80::/10')
 
     def test_ip_defined(self):
         if sys.version_info >= (3, 3):
-            from ipaddress import (IPv4Address, IPv6Address)
+            from ipaddress import (IPv4Address, IPv6Address, 
+                    IPv4Network, IPv6Network)
         else:
-            from ipaddr import (IPv4Address, IPv6Address)
+            from ipaddr import (IPv4Address, IPv6Address,
+                    IPv4Network, IPv6Network)
 
         self.assertRaises(IPDefinedError, Net, '192.168.0.1')
         self.assertRaises(IPDefinedError, Net, 'fe80::')
         self.assertRaises(IPDefinedError, Net, IPv4Address('192.168.0.1'))
         self.assertRaises(IPDefinedError, Net, IPv6Address('fe80::'))
+        self.assertRaises(IPDefinedError, Net, '192.168.0.0/16')
+        self.assertRaises(IPDefinedError, Net, 'fe80::/10')
+        self.assertRaises(IPDefinedError, Net, IPv4Network('192.168.0.0/16'))
```

Comment:
This is not building on Python 2.7. I think it has to do with your addition of unicode() which seems unnecessary.

Reply:
I fixed that, the problem was that the test used ipaddr.IPv4Network but now the net.py code check for ipaddress.IPv4Network. ipaddr has been superseded by ipaddress for both python3 and python2. It might be better to replace all the instances of ipaddr to ipaddress across all the files?